docs/source/markdown/podman-rmi.1.md

```diff
@@ -24,6 +24,10 @@ Remove all images in the local storage.
 
 This option will cause podman to remove all containers that are using the image before removing the image from the system.
 
+#### **--ignore**, **-i**
+
+Ignore if a specified image does not exist in the local storage and do not throw an error.
```

Comment:
```suggestion
If a specified image does not exist in the local storage, ignore the error and continue processing.
```

Reply:
"continue processing" isn't accurate. `rmi` always continues processing but throws errors at the end. With --ignore, such errors will not be reported.

Reply:
Rephrased but without the "continue" part :heavy_check_mark: